docs-source/content/samples/simple/elastic-stack/weblogic-domain/_index.md

```diff
@@ -0,0 +1,358 @@
+* Has volume mounts for the `fluentd-config` `ConfigMap` and the volume containing the domain logs
+
+**NOTE**: For brevity only the paths to the relevant configuration being added is shown.  A complete example of a domain definition is at the end of this document.
+
+Example: `kubectl edit domain bobs-bookstore -n bob` and add the following container definition.
+```yaml
+spec:
+  serverPod:
+    containers:
+    - args:
+      - -c
+      - /etc/fluent.conf
+      env:
+      - name: DOMAIN_UID
+        valueFrom:
+          fieldRef:
+            fieldPath: metadata.labels['weblogic.domainUID']
+      - name: SERVER_NAME
+        valueFrom:
+          fieldRef:
+            fieldPath: metadata.labels['weblogic.serverName']
+      - name: LOG_PATH
+        value: /scratch/logs/bobs-bookstore/$(SERVER_NAME).log
+      - name: FLUENTD_CONF
+        value: fluentd.conf
+      - name: FLUENT_ELASTICSEARCH_SED_DISABLE
+        value: "true"
+      - name: ELASTICSEARCH_HOST
+        valueFrom:
+          secretKeyRef:
+            key: elasticsearchhost
+            name: bobs-bookstore-weblogic-credentials
+      - name: ELASTICSEARCH_PORT
+        valueFrom:
+          secretKeyRef:
+            key: elasticsearchport
+            name: bobs-bookstore-weblogic-credentials
+      - name: ELASTICSEARCH_USER
+        valueFrom:
+          secretKeyRef:
+            key: elasticsearchuser
+            name: bobs-bookstore-weblogic-credentials
+            optional: true
+      - name: ELASTICSEARCH_PASSWORD
+        valueFrom:
+          secretKeyRef:
+            key: elasticsearchpassword
+            name: bobs-bookstore-weblogic-credentials
+            optional: true
+      image: fluent/fluentd-kubernetes-daemonset:v1.3.3-debian-elasticsearch-1.3
+      imagePullPolicy: IfNotPresent
+      name: fluentd
+      resources: {}
+      volumeMounts:
+      - mountPath: /fluentd/etc/fluentd.conf
+        name: fluentd-config-volume
+        subPath: fluentd.conf
+      - mountPath: /scratch
+        name: weblogic-domain-storage-volume
+```
+
+#### Verify logs are exported to Elasticsearch
+
+After the Administration Server and Managed Server pods have started with all the changes described above, the logs should now be sent to Elasticsearch.
+
+You can check if the `fluentd` container is successfully tailing the log by executing a command like `kubectl logs -f bobs-bookstore-admin-server -n bob fluentd`.  The log output should look similar to this:
+```text
+2019-10-01 16:23:44 +0000 [info]: #0 starting fluentd worker pid=13 ppid=9 worker=0
+2019-10-01 16:23:44 +0000 [warn]: #0 /scratch/logs/bobs-bookstore/managed-server1.log not found. Continuing without tailing it.
+2019-10-01 16:23:44 +0000 [info]: #0 fluentd worker is now running worker=0
+2019-10-01 16:24:01 +0000 [info]: #0 following tail of /scratch/logs/bobs-bookstore/managed-server1.log
+```
+
+Connect to Kibana you will see an index created for the `domainUID`.
```

Comment:
Connect to Kibana you will see -> When you connect to Kibana, you will see (Just wanted you to add a comma after Kibana; you must have thought I was removing the first two words; I wasn't).